src/components/OcTag.vue

```diff
@@ -19,6 +19,15 @@ export default {
       required: false,
       default: "span",
       validator: (type) => type.match(/(span|button|router-link|a)/)
+    },
+
+    /**
+     * Target of the router link
+     */
+    to: {
```

Comment:
Why is it needed, when it is added to the root component anyway?

Reply:
TBH I am not sure but if it's not passed, then in ocis-web it doesn't render with console errors about missing required `to` prop and the whole path is then undefined :shug:

Reply:
Ok 🤷 😅